lib/ifupdown.sh.in

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 
+# This script should be symlinked into /etc/network/ip-pre-up.d/ and
```

Comment:
@PaulSD 
out of curiosity ; 
are you using some sort of automatic formatting tool, or is this manual ?

if automatic, i'd like to try to include it in the Travis CI build ;